solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -717,18 +717,27 @@ contract RandomBeacon is Ownable {
 
     /// @notice Creates a new relay entry.
     /// @param entry Group BLS signature over the previous entry.
-    function submitRelayEntry(bytes calldata entry) external {
+    /// @param activeMembers Active group member ids array.
+    function submitRelayEntry(
+        bytes calldata entry,
+        uint32[] calldata activeMembers
```

Comment:
100% agreed. Adding one more point: in case of `submitRelayEntry`, `reportRelayEntryTimeout`, and similar, the additional parameter should be named just `groupMembers`.

Would love us to first asses the impact before we actually decide on this change. This code was already audited.

Reply:
Yea, makes sense. All the points 1-4 seem correct. I've renamed these vars here: 4c847d3